src/packet.ts

```diff
@@ -121,6 +121,27 @@ export class Packet {
     }
   }
 
+  public get payload(): Record<number, number> {
+    return this.privatePayload instanceof Buffer
+      ? objectify(this.privatePayload)
+      : this.privatePayload;
+  }
+
+  public get payloadAsBuffer(): Buffer {
+    return this.privatePayload instanceof Buffer ? this.privatePayload : null;
+  }
+
+  public get payloadAsRawArray(): Array<number> {
```

Comment:
I'm happy with the code, but I'm not sure if we need this method. To recevice an array, users can use:
```js
const payloadAsRawArray = [...packet.payloadAsBuffer]
```

Reply:
This works for >ES5 only. Your call how to do it :)

Reply:
That should be fine since this library only supports node.js v10+, and the 'spread' syntax has existed since node.js v5 ([source](https://node.green/#ES2015-syntax-spread-syntax-for-iterable-objects-with-arrays--in-function-calls)).

I'd say we should remove `payloadAsRawArray()` and let users convert the Buffer or object if neither option suits them. 

If you make that change I can merge this :)

Reply:
Done in beb6014